### PR TITLE
(FE):  code review change account link

### LINKED ISSUE #359: Fix Broken links
links to eosio documentation are broken 

https://guide.eoscostarica.io/docs/eos-learn/accounts-and-permissions/

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolves #359 

### Steps to test
1. Run `yarn build`
2. Go to `localhost:3000`
3. Go to Developer Guidelines > Learn EOSIO > Accounts and Permissions

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/eos-learn/accounts-and-permissions.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Any given account can define a mapping between any of its named permissions and
 
 In other words each account's permission can be linked to an authority table used to determine whether a given action authorization can be satisfied
 
-To get more information about these concepts, see [accounts and permissions documentation](https://developers.eos.io/welcome/latest/protocol/accounts_and_permissions).
+To get more information about these concepts, see [accounts and permissions documentation](https://developers.eos.io/welcome/v2.0/protocol-guides/accounts_and_permissions).
```
Comment (leisterfrancisco) [inline docs/eos-learn/accounts-and-permissions.md:31]: Nice catch.

Use the latest version: https://developers.eos.io/welcome/latest/protocol-guides/accounts_and_permissions

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

File: docs/eos-learn/accounts-and-permissions.md (hunk 1 above)
Comment (xavier506) [reply docs/eos-learn/accounts-and-permissions.md:31]: @hanakoori01

**Review by xavier506 — APPROVED** [review]
LGTM

**Comment by hanakoori01** [comment]
Please do not merge this PR, I'm solving some problems with my repository.
@xavier506

**Pull request closed without merging**